Frontend/src/Payment/Checkout.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import Image from "../assets/location.webp"
+import Slot from "../assets/slot.webp"
+import paymentImage from "../assets/payment.webp"
+import {useState} from "react"
+import { useNavigate } from "react-router-dom";
+
+
+export function CheckOut({amount}){
+    const [number, setNumber] = useState(9999999999)
+    const [address, setaddress] = useState("sabesh")
+    const [slot, setslot] = useState("boomer")
+    const navigate = useNavigate()
+
+    let AddressClass = address ? 'CheckOut_box' : 'CheckOut_box_Inactive'
+    let slotClass = address ? 'CheckOut_box' : 'CheckOut_box_Inactive'
+
+    function handleProceed(){
+        navigate("/payment")
+    }
+
+    return(
+        <div className = "CheckOut">
+            <div className="CheckOut_Header">
+                <h1>Checkout</h1>
```

Comment:
use constants as in https://github.com/AYS-At-Your-Service/AYS/pull/13/files#diff-c2c392d453cc978fcfed88aae84cdeb1b2ce75f60d0bb32f26cc6e821ab39a7b